doc/sources.rst

```diff
@@ -646,3 +648,118 @@ Example:
   debug_source:
     type: debug
 
+
+.. _ogcapitiles_label:
+
+OGC API Tiles
+"""""""""""""
+
+.. versionadded:: 5.1.0
```

Comment:
As we do have version specific documentations, you can omit the `versionadded`, this is not needed anymore.